ios/Classes/HyperWebviewFlutterPlugin.m

```diff
@@ -0,0 +1,106 @@
+#import "HyperWebviewFlutterPlugin.h"
+
+@interface HyperWebviewFlutterPlugin()
+
+@property (nonatomic, strong) FlutterMethodChannel* dartChannel;
+@property (nonatomic, strong) FlutterMethodChannel* nativeChannel;
+
+@end
+
+const int OPENAPPS_REQUEST_CODE = 19;
+const int GET_RESOURCE_NAME = 453;
+const int CAN_OPEN_APP_CODE = 789;
+@implementation HyperWebviewFlutterPlugin
+
+
++ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
+    HyperWebviewFlutterPlugin* instance = [[HyperWebviewFlutterPlugin alloc] init];
+    instance.dartChannel=[FlutterMethodChannel methodChannelWithName:@"DartChannel" binaryMessenger:[registrar messenger]];
+    FlutterMethodChannel* channel = [FlutterMethodChannel methodChannelWithName:@"NativeChannel" binaryMessenger:[registrar messenger]];
+    instance.nativeChannel=channel;
+    [registrar addMethodCallDelegate:instance channel:channel];
+}
+
+- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
+  if ([@"openApp" isEqualToString:call.method]) {
+      [self openApp:[call.arguments objectAtIndex:1]];
+  } if ([@"getResourceByName" isEqualToString:call.method]) {
+      NSString* resourceVal = [self getResourceByName:[call.arguments firstObject]];
+      NSDictionary *resource = @{
+          @"requestCode" : @(GET_RESOURCE_NAME),
+          @"payload" : resourceVal
+      };
+      [self.dartChannel invokeMethod:@"onActivityResult" arguments:resource];
+  } if ([@"canOpenApp" isEqualToString:call.method]) {
+      [self canOpenApp:[call.arguments firstObject]];
+  } else {
+      result(FlutterMethodNotImplemented);
+  }
+    result(@"true");
+}
+
+/**
+ To Check if an app can be opened by current app
+
+ @param payload App URL
+ */
+- (void)canOpenApp:(NSString*)payload{
+    //Note: canOpen will fail even if app is present in the device if app urls are not
+    //added to info.plist under LSApplicationQueriesSchemes as an array of allowed urls.
+    NSURL *appURL = [NSURL URLWithString:[HyperWebviewFlutterPlugin base64DecodedStringFromString:payload]];
+    BOOL status = [[UIApplication sharedApplication] canOpenURL:appURL];
+    NSString* resPayload = [NSString stringWithFormat:@"{\"result\":\"%@\",\"app\":\"%@\"}", status?@"1":@"0", appURL];
```

Comment:
let's create an disctionary and convert to String

Reply:
@Subhash703 - to take care in next PR if still open